src/main/java/com/greghaskins/spectrum/AbstractSupplyingHook.java

```diff
@@ -6,8 +6,9 @@
 import org.junit.runner.notification.RunNotifier;
 
 /**
- * A base class for supplying hooks to use. Override before or after. Return the singleton
- * value from the before method.
+ * A base class for {@link SupplyingHook hooks that supply a value}.
+ *
+ * <p>Override {@link #before} or {@link #after}. Return the singleton value from the before method.
```

Comment:
The existing opening sentence read ambiguously: it's not for supplying actual hooks, it's for hooks that supply a value.

While I was there I hyperlinked to the referenced methods to help them "pop" and to call out that they're not just words but method names.